src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -59,6 +60,7 @@ class RendererOpenGL : public RendererBase {
     void DrawScreens();
     void DrawSingleScreenRotated(const ScreenInfo& screen_info, float x, float y, float w, float h);
     void UpdateFramerate();
+    void FrameLimiter(float average_frame_time, int frame_limit);
```

Comment:
i have tried float multiplying average_frame_time by 100 and casting it as uint32, and dividing it in the function by 100 as float. does not work, and other different combination(making sure to raise before passing and reduce in function). i think leaving it as (float ,uint32_t) is good for now.